### PR TITLE
docs: Standardize the README

### PR DESCRIPTION
**Pull request by gnumoksha**
What:
Standardizes the README file providing a common language and an implementation link.

Why:
There are differences between PSR's READMEs in regarding language and the lacking of implementations references. So I've updated all READMEs using the https://github.com/php-fig/http-factory as base.

### CODE REVIEW

**Review by Jean85 — APPROVED** [review]
This LGTM, it's only an aesthetic change. I would like to wait for @mnapoli & @moufmouf approval nonetheless.

**Review by mnapoli — APPROVED** [review]
:+1: for me as well

**Pull request merged**
